src/gen-charts.ts

```diff
@@ -1563,6 +1563,9 @@ function makeChartType(chartType: CHART_NAME, data: IOptsChartData[], opts: ICha
 			strXml += '<c:' + chartType + 'Chart>'
 			strXml += '  <c:varyColors val="1"/>'
 			strXml += '<c:ser>'
+			if (opts.chartPieExplosion) {
```

Comment:
we'll need to do a couple of safety checks before passing it to the xml i.e. is it a valid number?

Reply:
is the following fine?
```js
if (opts.chartPieExplosion && Number.isInteger(opts.chartPieExplosion)) {
```

Reply:
Do we support floats? And is there a min or max limit? I've seen some PPTX errors before because the number were out of range.

Reply:
and Explosion needs to be an integer value according to [c-rex](https://c-rex.net/projects/samples/ooxml/e1/Part4/OOXML_P4_DOCX_explosion_topic_ID0EAIBQB.html?hl=explosion)

I don't think we should be overly defensive here. The values which Pitch can produce are quite narrow (0 - 10).